src/quo2/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,52 @@
+(ns quo2.components.avatars.wallet-user-avatar.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.wallet-user-avatar.style :as style]
+            utils.string))
+
+(def properties
+  {:size-20 {:size        20
+             :font-size   :label
+             :font-weight :medium}
+   :size-24 {:size        24
+             :font-size   :label
+             :font-weight :semi-bold}
+   :size-32 {:size        32
+             :font-size   :paragraph-2
+             :font-weight :semi-bold}
+   :size-48 {:size        48
+             :font-size   :paragraph-1
+             :font-weight :semi-bold}
+   :size-64 {:size        64
+             :font-size   :paragraph-1
+             :font-weight :medium}
+   :size-80 {:size        80
+             :font-size   :heading-1
+             :font-weight :medium}})
+
+(defn- view-internal
+  "Options:
+
+  :full-name - string (default: nil) - used to generate initials
+  :color  - keyword (default: nil) - color of the avatar
+  :size   - keyword (default: last element of properties object) - size of the
+  avatar"
+  [{:keys [full-name customization-color size theme]
+    :or   {size (last (keys properties))}}]
+  (let [circle-size  (:size (size properties))
+        small?       (= size :size-20)
+        circle-color (style/circle-color customization-color)
```

Comment:
this circle color is a bit strange, you are calling a function from style file only to pass it into a method in the style file. 
consider doing this inside `style/container` function directly.
e.g ` {:style (style/container circle-size customization-color)}`


```
(defn- circle-color
  [customization-color]
  (colors/custom-color customization-color 50 20))

(defn container
  [circle-size customization-color]
  {:width            circle-size
   :height           circle-size
   :border-radius    circle-size
   :text-align       :center
   :justify-content  :center
   :align-items      :center
   :background-color (circle-color customization-color)})

```

Reply:
wow thanks thats something i missed